jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/ConnectionMetaData.java

```diff
@@ -52,9 +52,7 @@ default boolean isSecure()
 
     /**
      * @return whether the functionality of pushing resources is supported
-     * @deprecated in favour of 103 Early Hints
      */
-    @Deprecated(since = "12.0.1")
     default boolean isPushSupported()
```

Comment:
Unrelated change??

Reply:
Note entirely. I was fixing the push cookie handling and realized this core method should not be deprecated